gen/__init__.py

```diff
@@ -491,6 +491,7 @@ def get_dcosconfig_source_target_and_templates(
         'dcos_image_commit',
         'package_ids',
         'template_filenames',
+        'enable_windows_agents',
```

Comment:
targets are collected from the variables used in templates, plus a list of variables used outside of the templates.
I added `enable_windows_agents` as a base target, to validate calico_network_cidr when windows is not enabled.